ParkBlazerApp/src/components/map.tsx

```diff
@@ -142,6 +150,21 @@ export default function Map({ onUpdateList, onLocationMarkerUpdate }: any) {
               <br></br>
               <IonText><strong>Lastwagen:</strong> <IonIcon icon={selectedSpot.type_truck ? checkmark : close} /></IonText>
             </IonCard>
+            {isPrivateModal && (
+              <>
+                <IonCard>
+                  <IonText><strong>Preis:</strong> {selectedSpot.price_per_hour} €</IonText>
+                </IonCard>
+                {selectedSpot.availability_start_date && selectedSpot.availability_end_date && (
+                  <IonCard>
+                  <IonText><strong>Verfügbarkeit:</strong></IonText>
+                  <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', padding: '20px' }}>
+                    <IonDatetime className="date-picker" value={selectedSpot?.availability_start_date} readonly={true} />
```

Comment:
Nur den Start Date zu zeigen, reicht nicht aus. Du kannst die ``min`` und ``max`` properties von IonDatetime benutzten, um den Zeitraum darzustellen. Außerdem kann man die Zeit Auswahl mit der propertie ``presentation='date'`` unsichtbar machen.
```suggestion
                    <IonDatetime className="date-picker" presentation='date' min={selectedSpot.availability_start_date} max={selectedSpot.availability_end_date} readonly={true} />
```